firefox-ios/Client/Assets/CC_Script/CC_Python_Update.py

```diff
@@ -67,16 +70,19 @@ def downloadTemporaryFileToCompare(file_info):
     except requests.exceptions.HTTPError as err:
         raise SystemExit(err)
 
+
```

Comment:
```suggestion
```
Q. do you need these spaces? Not sure if you added intentionally or it just got added

Reply:
Oh the formatter is being annoying. Let me fix that.